Packs/Logzio/Integrations/Logzio/Logzio.yml

```diff
@@ -0,0 +1,102 @@
+commonfields:
+  id: Logz.io
+  version: -1
+name: Logz.io
+display: Logz.io
+category: Analytics & SIEM
+description: Fetches incidents from Logz.io cloud SIEM to activate Demisto playbooks.
+configuration:
+- display: Fetch incidents.
+  name: isFetch
+  type: 8
+  required: false
+- display: Incident type
+  name: incidentType
+  type: 13
+  required: false
+- display: API token for Logz.io Security account
+  name: security_api_token
+  type: 4
+  required: false
+- display: API token for Logz.io Operations account
+  name: operational_api_token
+  type: 4
+  required: false
+- display: Region code of your Logz.io account
+  name: region
+  defaultvalue: us
+  type: 0
+  required: false
+- display: Filter by rule names (Apache Lucene syntax)
+  name: search
+  type: 0
+  required: false
+- display: Filter by rule severity
+  name: severities
+  type: 16
+  required: false
+  options:
+  - SEVERE
+  - HIGH
+  - MEDIUM
+  - LOW
+  - INFO
+- display: First-time retroactive fetch (e.g., 12 hours, 7 days)
+  name: fetch_time
+  defaultvalue: 1 hours
+  type: 0
+  required: false
+- display: Trust any certificate (not secure)
+  name: insecure
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  type: 8
+  required: false
+script:
+  script: ''
+  type: python
+  commands:
+  - name: logzio-search-logs
+    arguments:
+    - name: query
+      required: true
+      default: true
+      description: A string specifying the search query, written in Apache Lucene syntax. The default is '*'.
+      defaultValue: '*'
+    - name: size
+      description: An integer specifying the maximum number of results to return.
+    - name: from_time
+      description: Unix timestamp. Specifies the earliest timestamp to be returned by the query.
+    - name: to_time
+      description: Unix timestamp. Specifies the latest timestamp to be returned by the query.
+    description: Returns logs from your Logz.io operations account using an Apache Lucene query.
+    outputs:
+    - contextPath: Logzio.Logs.Count
+      description: Number of results
+      type: number
+    - contextPath: Logzio.Logs.Results
+      description: An array of the search results
+    description: Returns logs from your Logz.io Operations account by Lucene query
+  - name: logzio-get-logs-by-rule-id
+    arguments:
+    - name: id
+      required: true
+      description: Security rule ID in Logz.io. In Demisto, it appears under the field
+        'alertEventId'.
+    - name: size
+      default: true
+      description: An integer specifying the maximum number of results to return
+      defaultValue: "100"
+    outputs:
+    - contextPath: Logzio.Logs.Count
```

Comment:
The output in the code has changed, now you just output `Logzio.Logs`. Please update the outputs and regenerate documentation accordingly.

Also please change this (also in the code) to `Logzio.Result` to be consistent to other SIEM integrations (i.e. Splunk)